src/components/Audio.js

```diff
@@ -0,0 +1,20 @@
+import React, {useState} from 'react';
+import mute from '../icons/mute.svg';
+import unmute from '../icons/unmute.svg';
+
+function Audio() {
+  //switching icon from unmute to mute
+  const [toggle, setToggle]= useState(false);
+  const toggler =() => {
+    toggle ? setToggle(false):setToggle(true);
+  };
```

Comment:
These two consts are duplicated here and in App.js--I would store these in App.js instead and pass them down here.